index.html

```diff
@@ -45,9 +45,17 @@
             <span>Download CV</span>
         </a>
     </div>
+
+    <div class="hamburger" id="hamburger">
+        <i class="fas fa-bars"></i>
+    </div>
+
+    <!-- Close Icon for the mobile menu -->
+    <div class="close-menu" id="closeMenu">
+        <i class="fas fa-times"></i>
+    </div>
 </nav>
```

Comment:
Hey,
please add style for your hamburger.
e.g
/* hamburger icon starts*/
#menu {
  font-size: 3rem;
  cursor: pointer;
  color: rgb(24, 2, 63);
  display: none;
}
@media (max-width: 768px) {
  #menu {
    display: block;
  }
  header .navbar {
    position: fixed;
    top: 6.5rem;
    right: -120%;
    width: 75%;
    height: 100%;
    text-align: left;
    align-items: flex-start;
    background-color: #0e0f31;
  }
  header .navbar ul {
    flex-flow: column;
    padding: 1rem;
  }
  header .navbar ul li {
    text-align: center;
    width: 100%;
    margin: 1rem 0;
    border-radius: 0.5rem;
    width: 26rem;
  }
  header .navbar ul li a {
    display: block;
    padding: 1rem;
    text-align: left;
    color: #fff;
    font-size: 2rem;
  }
  header .navbar ul li a.active,
  header .navbar ul li a:hover {
    padding: 1rem;
    color: #fff;
    border-radius: 0.5rem;
    border-bottom: 0.5rem solid #011aff;
  }
  .fa-times {
    transform: rotate(180deg);
  }
  header .navbar.nav-toggle {
    right: 0;
  }
}
/* hamburger icon ends */
After adding it 

you will need to have something like this in your html
<div id="menu" class="fas fa-bars"></div>
        <nav class="navbar">
            <ul>
            <li><a class="active" href="#home">Home</a></li> 
            <li><a href="#about">About</a></li>
            <li><a href="#skills">Skills</a></li> 
            <li><a href="#education">Education</a></li>
            <li><a href="#experience">Experience</a></li>
            </ul>
        </nav>

Reply:
hi @Andro-crt-prog 
I’ve made the changes you requested regarding the style for the hamburger. Please let me know if it meets your expectations now.